src/firewall/core/fw_ipset.py

```diff
@@ -41,9 +41,8 @@ def __repr__(self):
     def cleanup(self):
         self._ipsets.clear()
 
-    def check_ipset(self, name):
-        if name not in self.get_ipsets():
-            raise FirewallError(errors.INVALID_IPSET, name)
+    def check_ipset(self, name, applied=False):
+        return self.get_ipset(name, applied=applied).name
```

Comment:
I don't think there are many callers (none?) of this now. Let's just remove it if we can.

Reply:
there is one... `check_applied()`.

I would rather remove `check_applied()`, and have them call `check_ipset(name, applied=True)`.

You said, you prefer multiple functions over arguments. I disagree, because the argument here restricts the check, that is, by setting any of those arguments, we check more. Without any arguments, it is just the most basic check, but with arguments, we restrict it further (must also be `applied`).